docs/getting-started.mdx

```diff
@@ -166,17 +166,24 @@ Psst... Android is a bit easier to set up, so we recommend starting with it!
 
       1. Open `ios/Runner.xcworkspace` in Xcode. 
       
-      2. Create a test target if you do not already have one via `File > New > Target...`
-         and select `UI Testing Bundle`. Change the `Product Name` to `RunnerUITests`. Make
-         sure `Target to be Tested` is set to `Runner` and language is set to `Objective-C`.
-         Select `Finish`.
+      2. Create a test target if you do not already have one (see the screenshot below
+         for the reference). Select `File > New > Target...` and select `UI Testing Bundle`.
+         Change the `Product Name` to `RunnerUITests`. Set the `Organization Identifier` 
+         to be the same as for the `Runner` (no matter if you app has flavors or not). 
+         For our example app, it's `com.example.MyApp` just as in the `pubspec.yaml` file. 
+         Make sure `Target to be Tested` is set to `Runner` and language is set to `Objective-C`.
+         Select `Finish`. 
+
+      ![Xcode iOS test target](/assets/ios_test_target.png)
 
       3. 2 files are created: `RunnerUITests.m` and `RunnerUITestsLaunchTests.m`.
-         Delete `RunnerUITestsLaunchTests.m` **through Xcode**.
+         Delete `RunnerUITestsLaunchTests.m` **through Xcode** by clicking on it and 
+         selecting `Move to Trash`.
 
       4. Make sure that the **iOS Deployment Target** of `RunnerUITests` within the
          **Build Settings** section is the same as `Runner`.
-         The minimum supported **iOS Deployment Target** is `11.0`.
+         The minimum supported **iOS Deployment Target** is `11.0`. For the example app,
+         we set it to `13.0`.
```

Comment:
Explain why - because depencies require us to.
Also link to example app. So one can easily hover and click.